### ISSUE TITLE
[BUG] My server crashes when placing world lock/breaking world lock!

### ISSUE THREAD

**Issue by coderfixer** [GrowtopiaNoobs/GrowtopiaServer#1307]
My server is crashing at the moment of breaking or placing world lock. I've been trying to fix it but did it without success. Is there any way to fix it? Because it's really hard for me and i'm in learning state of c++ and i would like someone to help me with this problem.

**Comment by cmdocmd** [comment]
because you are using a leaked cpp use growtopia noobs cpp doesnt have this bug

**Comment by coderfixer** [comment]
no, its not about the cpp. its something in the files i think. i think its Coredata or items.dat

**Comment by GuckTubeYT** [comment]
> no, its not about the cpp. its something in the files i think. i think its Coredata or items.dat

maybe at cpp is wrong

**Comment by rama-adi** [comment]
> I'm in learning state of c++
That's the problem. C++ is a low-level language and is difficult to learn. I suggest that you learn C++ first, then tackle projects like these. It's not a "one-size-fits-all" project.

**Comment by xXAbood** [comment]
> because you are using a leaked cpp use growtopia noobs cpp doesnt have this bug

me when i include a cpp in everything
